server/repository/src/migrations/v1_07_00/changelog_add_name_link_id.rs

```diff
@@ -52,20 +53,33 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
 
         -- Adding changelog.name_link_id
         PRAGMA foreign_keys = OFF;
-        ALTER TABLE changelog
-        ADD COLUMN name_link_id TEXT REFERENCES name_link(id);
-        
-        UPDATE changelog
-        SET name_link_id = name_id;
+
+        CREATE TABLE tmp_changelog AS SELECT * FROM changelog;
+        DROP TABLE changelog;
```

Comment:
Can't this just be a table rename? 🤔

Reply:
If I rename it seems like the triggers get moved with the table, but that does make me worry that triggers might get screwed up here...